src/header/header.scss

```diff
@@ -190,181 +136,191 @@
-          @include themed {
-            background-color: t(iui-color-background-1);
-          }
-        }
       }
     }
-  }
 
-  .iui-header-button-icon {
-    margin: 0 $iui-l;
-    @include themed {
-      fill: t(iui-icons-color-actionable);
-    }
-  }
+    .iui-slim {
+      .iui-header-brand-icon {
+        width: $iui-icons-default;
+        height: $iui-icons-default;
+        margin: 0 $iui-l + $iui-xs;
+      }
 
-  > .iui-label {
-    margin-right: $iui-sm;
-    font-size: $iui-font-size-leading;
-    @media (prefers-reduced-motion: no-preference) {
-      transition: font-size $iui-speed-fast ease-out;
+      .iui-header-brand-label {
+        font-size: $iui-font-size;
+      }
     }
   }
 }
 
-@mixin iui-header-button-focus {
-  &:focus {
-    box-shadow: none;
-    outline-offset: -1px;
-
-    @include themed {
-      outline: 1px solid t(iui-color-foreground-primary);
-    }
-  }
-
-  &:focus:not(:focus-visible) {
-    outline: 0;
-  }
-}
-
-@mixin iui-header-button {
+@mixin iui-header-breadcrumbs-button {
+  @include iui-button;
+  @include iui-button-borderless;
   @include iui-button-size(large, borderless);
-
+  @include iui-header-button-reset;
   gap: 0;
   max-width: 25vw;
-  height: 100%;
-  border-radius: 0;
   overflow: hidden;
   padding-right: $iui-s;
 
-  @include iui-header-button-focus;
+  &-name,
+  &-description {
+    overflow: hidden;
+    text-overflow: ellipsis;
+    transition: color $iui-speed-fast ease-out;
+  }
+
+  &-description {
+    height: $iui-baseline * 2;
+    @media (prefers-reduced-motion: no-preference) {
+      transition: height $iui-speed-fast ease-out;
+    }
+    @include themed {
+      opacity: t(iui-opacity-3);
+    }
+  }
 
   > .iui-button-label {
     text-align: left;
     font-size: $iui-font-size;
     overflow: hidden;
 
-    > div {
-      overflow: hidden;
-      text-overflow: ellipsis;
-    }
-
-    .iui-description {
-      height: $iui-baseline * 2;
-      @media (prefers-reduced-motion: no-preference) {
-        transition: all $iui-speed-fast ease;
-      }
-      @include themed {
-        opacity: t(iui-opacity-3);
-      }
+    @media (max-width: 768px) {
+      margin-left: 0;
     }
   }
 
   > .iui-button-icon {
     flex-shrink: 0;
+    box-sizing: content-box;
   }
 
   > * + * {
     margin-left: $iui-sm;
   }
 
   .iui-header-button-icon {
+    display: inline-flex;
     padding: $iui-xs;
+    box-sizing: content-box;
     @include iui-header-button-icon($iui-icons-default, $iui-border-radius);
     @include themed {
       background-color: t(iui-color-background-3);
       fill: t(iui-icons-color);
     }
+
+    @media (max-width: 768px) {
+      display: none;
+    }
+  }
+
+  img.iui-header-button-icon {
+    padding: 0;
+    width: auto;
+    height: $iui-icons-default + ($iui-xs * 2);
   }
 
   &.iui-active {
     @include iui-header-button-active;
   }
 
-  &:disabled {
+  &.iui-disabled,
+  &:disabled,
```

Comment:
I think hover is broken...
Can we add in demo disabled states too? Same as with split buttons, so we coudl test easier

Reply:
Just saw that now too, appears the right side of a disabled split button is broken.  Looking into it.

Reply:
Hover is still broken
![image](https://user-images.githubusercontent.com/9084735/156257475-718e7705-3d8e-41d4-857c-794576b8472c.png)

I've added the disabled toggle to all sections in [`75c937e`](https://github.com/iTwin/iTwinUI/pull/463/commits/75c937e2f1b510876ec5a3fde6311ef1433fd7af).

Reply:
Hover is now fixed.